documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,84 @@
+# Get Bootchart from a Clear Container#
```

Comment:
Looks like you've been caught by what might be a github markdown behaviour change - if you click on the "View" button for this file, you'll see that it isn't rendering correctly. You (now?) need to leave a space before the `#`.